includes/abstracts/abstract-wc-order.php

```diff
@@ -446,6 +446,45 @@ public function get_subtotal() {
 		return apply_filters( 'woocommerce_order_get_subtotal', (float) $subtotal, $this );
 	}
 
+	/**
+	 * Get the total amount of tax for line items.
+	 *
+	 * @return float
+	 */
+	public function get_subtotal_tax() {
+		$total = 0;
+		foreach ( $this->get_items() as $item ) {
+			$total += wc_remove_number_precision( self::round_line_tax( wc_add_number_precision( $item->get_subtotal_tax() ) ) );
```

Comment:
If possible, can we get rid of `wc_remove_number_precision` and `wc_add_number_precision`. I am investigating to see if we can deprecate and remove usages of these methods as follow up to #25800. These methods have a side effect that they round by default -- that said, I am not sure if I will go ahead with deprecation because I still don't fully understand the reasoning behind these.
Maybe use `self::round_line_tax( ..., false )` instead for now?